index.html

```diff
@@ -164,6 +164,30 @@ <h3>Skills</h3>
                     <div class="bulb"></div>
                 </div>
             </div>
+        </section>
+    <section class="section-4">
+        <div class="mobile-form-wrapper">
+            <h1 class="get-started">Get Started</h1>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus architecto, atque laudantium autem vel explicabo soluta quos fugit quod distinctio. Ea ullam ut facere numquam minima dignissimos? Mollitia, esse iure.</p>
+        <form action="https://formspree.io/f/xoqzvkpg" method="post">
+            <label for="Name"><input type="text" placeholder="Your name" maxlength="30" class="name"></label>
+            <label for="Email"><input type="email" placeholder="Your email" class="email"></label>
+            <label for="Message"><textarea type="text" name="" maxlength="500" class="message" placeholder="Enter your message here.."></textarea></label>
```

Comment:
Good job! But let's add `required` to all of our input and textarea tag to complete our project requirements:
<img width="546" alt="Screenshot 2023-01-19 at 8 27 41 PM" src="https://user-images.githubusercontent.com/25887253/213517510-e797a105-5a62-4b17-9c11-2067b20f4709.png">

If you try to submit a form, you will have the same warnings from formspree.